src/scenes/Home/Home.js

```diff
@@ -12,9 +13,25 @@ class Home extends PureComponent {
   state = {
     questionPosition: 0,
     answersUser: defaultAnswers,
+    // used if i can access to api amazon
+    querySearch: '',
   }
 
   next = () => {
+    // boucle sur les reponses, utilisé pour les checkbox
+    for (let x = 0; x < this.state.answersUser[this.state.questionPosition].length; x += 1) {
```

Comment:
Je pense qu'on peut faire ça de manière plus propre, avec moins de for. Là c'est compliqué à comprendre ce que tu veux faire.

Reply:
@cdenys tu devais pas faire juste avec des strings ?

Reply:
yes, je vais simplifier le json pour mettre des strings à la place d'obj.